main-cookbooks/teracy-dev/recipes/docker.rb

```diff
@@ -65,6 +65,16 @@ def get_docker_compose_autocomplete_url
     "https://raw.githubusercontent.com/docker/compose/#{get_docker_compose_release}/contrib/completion/bash/docker-compose"
 end
 
+def install_bash_completion
+  bash 'check exist and install bash-completion' do
```

Comment:
Should create a new recipe for this task. 
`apt-get` should have `--reinstall -f -y` ( just in case package was broken and we want to make sure it will install no matter what )